lib/dialects/oracle/query/compiler.js

```diff
@@ -78,7 +77,7 @@ class QueryCompiler_Oracle extends QueryCompiler {
 
     const sql = {};
 
-    if (isString(insertData)) {
```

Comment:
can you extract this to an internal isString function as well, so that it would be easy for us to make it more robust if needed?